src/js/index.js

```diff
@@ -11,30 +11,32 @@ var model = require('./model');
 model.init();
 
 // Initialize components
-var Sidebars = require('./components');
+var ui = require('./components');
 
 var g = model.Scene.child('marks.group'),
+    g2 = g.child('marks.group'),
+    g3 = g2.child('marks.group'),
```

Comment:
from the functional test:
When you add groups it shows up in the editor layers. Do we want the g3 group for rects to be nested under g2?  Wouldn't we want it all in one group like before?  Was this just test code to make sure things were working?

![pipeline](https://cloud.githubusercontent.com/assets/607541/13888182/488895cc-ecfd-11e5-8125-1b8c16f21565.gif)